src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -105,6 +105,13 @@ class HuiGenericEntityRow extends LitElement {
                     .datetime=${stateObj.last_changed}
                   ></ha-relative-time>
                 `
+              : this.config.secondary_info === "last-triggered"
+              ? html`
+                  <ha-relative-time
+                    .hass=${this.hass}
+                    .datetime=${stateObj.attributes.last_triggered}
```

Comment:
`last-triggered` is not available on every entity, you should check for that.

Reply:
I understand it is only available on automation and script, (which is why I would need this and love to add it to the secondary-info).

I dont know how to test for that though, thought the final 
```
:""
```
safeguarded that, showing nothing in that case? Please suggest what could be used for this scenario?